src/app/games/epaminondas/EpaminondasTutorial.ts

```diff
@@ -104,13 +88,7 @@ export class EpaminondasTutorial extends Tutorial {
         ),
         TutorialStep.fromMove(
             $localize`Captures`,
-            $localize`In order to capture pieces of the opponent:
-        <ol>
-            <li>It must be aligned with the phalanx you are moving.</li>
-            <li>It must be strictly shorter than your phalanx.</li>
-            <li>The first piece of your phalanx should land on the first piece of the opponent's phalanx that you want to capture.</li>
-        </ol><br/>
-        You're playing Dark, capture a phalanx.`,
+            $localize`In order to capture pieces of the opponent:<ol><li>It must be aligned with the phalanx you are moving.</li><li>It must be strictly shorter than your phalanx.</li><li>The first piece of your phalanx should land on the first piece of the opponent's phalanx that you want to capture.</li></ol><br/>You're playing Dark, capture a phalanx.`,
```

Comment:
it must -> they must pour le premier
pour le second, c'pas correct comme formulation mais je sais pas comment rendre ça bien. Peut-être juste dire "Pour capturer une phalanx adverse" comme première phrase et garder la liste identique